tests/functional/spec/skip_conditions.spec.js

```diff
@@ -1,4 +1,4 @@
-const helpers = require('../helpers');
+/*const helpers = require('../helpers');
```

Comment:
Intentional?

Reply:
Ah, I had forgotten about this. Commented out to get all the tests running, I'll sort this out now.